<?xml version="1.0" encoding="UTF-8"?><testrun name="FakultasEndpointTest" project="akademik-api" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="id.its.akademik.endpoint.test.FakultasEndpointTest" time="2.267">
    <testcase name="canGetFakultas" classname="id.its.akademik.endpoint.test.FakultasEndpointTest" time="2.267">
      <error>org.springframework.dao.InvalidDataAccessApiUsageException: WRONGTYPE Operation against a key holding the wrong kind of value; nested exception is redis.clients.jedis.exceptions.JedisDataException: WRONGTYPE Operation against a key holding the wrong kind of value&#13;
	at org.springframework.data.redis.connection.jedis.JedisExceptionConverter.convert(JedisExceptionConverter.java:44)&#13;
	at org.springframework.data.redis.connection.jedis.JedisExceptionConverter.convert(JedisExceptionConverter.java:36)&#13;
	at org.springframework.data.redis.PassThroughExceptionTranslationStrategy.translate(PassThroughExceptionTranslationStrategy.java:37)&#13;
	at org.springframework.data.redis.FallbackExceptionTranslationStrategy.translate(FallbackExceptionTranslationStrategy.java:37)&#13;
	at org.springframework.data.redis.connection.jedis.JedisConnection.convertJedisAccessException(JedisConnection.java:210)&#13;
	at org.springframework.data.redis.connection.jedis.JedisConnection.rPush(JedisConnection.java:1528)&#13;
	at org.springframework.data.redis.core.DefaultListOperations$14.doInRedis(DefaultListOperations.java:187)&#13;
	at org.springframework.data.redis.core.DefaultListOperations$14.doInRedis(DefaultListOperations.java:184)&#13;
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:191)&#13;
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:153)&#13;
	at org.springframework.data.redis.core.AbstractOperations.execute(AbstractOperations.java:86)&#13;
	at org.springframework.data.redis.core.DefaultListOperations.rightPush(DefaultListOperations.java:184)&#13;
	at id.its.akademik.dao.jdbc.JdbcKelembagaanDao.getProdiAjar(JdbcKelembagaanDao.java:113)&#13;
	at id.its.akademik.endpoint.ProdiEndpoint.getProdiAjar(ProdiEndpoint.java:75)&#13;
	at id.its.akademik.endpoint.test.FakultasEndpointTest.canGetFakultas(FakultasEndpointTest.java:22)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:83)&#13;
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)&#13;
Caused by: redis.clients.jedis.exceptions.JedisDataException: WRONGTYPE Operation against a key holding the wrong kind of value&#13;
	at redis.clients.jedis.Protocol.processError(Protocol.java:113)&#13;
	at redis.clients.jedis.Protocol.process(Protocol.java:138)&#13;
	at redis.clients.jedis.Protocol.read(Protocol.java:192)&#13;
	at redis.clients.jedis.Connection.readProtocolWithCheckingBroken(Connection.java:282)&#13;
	at redis.clients.jedis.Connection.getIntegerReply(Connection.java:207)&#13;
	at redis.clients.jedis.BinaryJedis.rpush(BinaryJedis.java:890)&#13;
	at org.springframework.data.redis.connection.jedis.JedisConnection.rPush(JedisConnection.java:1526)&#13;
	... 37 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
